README.md

```diff
@@ -30,7 +30,7 @@ In addition to the Elixir config files, the V3 API allows runtime configuration
 | `MBTA_GTFS_URL` | `https://cdn.mbta.com/MBTA_GTFS.zip` | URL for the GTFS .zip file. |
 | `ALERTS_URL` | `https://cdn.mbta.com/realtime/Alerts_enhanced.json` | URL for the Alerts. Can be either a JSON or Protobuf file.
 | `MBTA_TRIP_SOURCE` | `https://cdn.mbta.com/realtime/TripUpdates_enhanced.json` | URL for the TripUpdates. Can be either a JSON or Protobuf file. |
-| `MBTA_VEHICLE_SOURCE ` | `https://cdn.mbta.com/realtime/VehiclePositions.pb` | URL for the VehiclePositions. Must be a Protobuf file. |
+| `MBTA_VEHICLE_SOURCE ` | `https://cdn.mbta.com/realtime/VehiclePositions_enhanced.json` | URL for the VehiclePositions. Must be a JSON file. |
```

Comment:
Can we support both? Otherwise it'll be hard/impossible to deploy since we can't do a configuration change and a code change at the same time.

Reply:
What we've done in the past is pattern match the incoming data:
```
def parse(body)
def parse("{" <> _ = body), do: parse_json(body)
def parse(body), do: parse_protobuf(body)
```